server/service/src/sync/translations/report.rs

```diff
@@ -121,6 +121,9 @@ impl SyncTranslation for ReportTranslation {
             comment,
             sub_context,
             argument_schema_id,
+            is_custom: todo!(),
```

Comment:
THese should have defaults now right? Which PR is that in?

Reply:
You're right - I missed this one, thank you James.

It might be in another higher stacked PR, but I am thinking should be in all of them.